### PR TITLE
Radiation changes

### PR DESCRIPTION
**Pull request by ghost**
- Atoms now have a proc, rad_act(). This accepts a number (i.e. severity) and can be used to generalize radiation effects on different objects
- There is a new proc, radiation_pulse(). It's like an EMP pulse, but with radiation.
- The mob proc, irradiate(), has been removed and moved to the mob's rad_act().
- Most radiation pulses have been replaced with the new radiation_pulse() proc.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Go work on vampires and redoing cult you promised

**Comment by Xhuis** [comment]
sssh it's okay

**Comment by Xhuis** [comment]
Squashed commits. This was pretty easy to make, and I figured it would improve code quality. This is actually merge-ready unless anyone has problems with it.

**Comment by AlexCrimson** [comment]
Xhuis the half-finished. It could be a title or something.

**Comment by Xhuis** [comment]
But this is completely done

File: config/admins.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
-Optimumtact = Host
+Xhuis = Host
```
Comment (Fox-McCloud) [inline config/admins.txt]: Hue

**Comment by Xhuis** [comment]
shit i forgot

**Comment by Xhuis** [comment]
YOU SAW NOTHING

**Comment by Razharas** [comment]
Looks nice

File: code/game/objects/radiation.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		var/distance = get_dist(epicenter, T)
+		if(distance < 0)
+			distance = 0
+		if(distance < heavy_range)
+			T.rad_act(severity)
+		else if(distance == heavy_range)
+			if(prob(50))
+				T.rad_act(severity)
+			else
+				T.rad_act(severity / 2)
```
Comment (RemieRichards) [inline code/game/objects/radiation.dm]: really unhappy these ```/ 2```'s got in. given this is a proc that will affect a LOT of atoms, that's a lot of iterations, so doing the division is bad.

Comment (duncathan) [inline code/game/objects/radiation.dm]: We should make the whole "multiply by the reciprocal" thing part of the code standards page.

Comment (RemieRichards) [inline code/game/objects/radiation.dm]: Yes, especially in cases such as this.

**Pull request merged**